web/packages/teleterm/src/ui/AppUpdater/WidgetView.tsx

```diff
@@ -54,6 +53,7 @@ export function WidgetView(props: {
   updateEvent: AppUpdateEvent;
   platform: Platform;
   clusterGetter: ClusterGetter;
+  mx?: string | number;
```

Comment:
Might be overkill for this but in Identity Security when I want to pass style properties to children components I try to extend either the props of the child component (`BoxProps`, etc) or the style props (`MarginProps` in this case), collecting them in `...rest` and passing `{...rest}` down, so I'm not redefining style types and it is more flexible in the future

Just a thought, feel free to ignore